packages/api/endpoints/logs.js

```diff
@@ -40,8 +40,10 @@ function handler(event, context) {
     if (event.httpMethod === 'GET' && event.resource === '/stats/logs') {
       return count(event, cb);
     }
-    else if (event.httpMethod === 'GET' && event.pathParameters.executionName) {
-      return get(event, cb);
+    else if (event.httpMethod === 'GET' && event.pathParameters) {
```

Comment:
Maybe we could use lodash.get:
`else if (event.httpMethod === 'GET' && get(event, pathParameters.executionName) {`
If the event.pathParameters is defined but executionName doesn't exist, what does the api return?

Reply:
Two things, at the risk of being pedantic. In this instance, what is the benefit of using lodash's `get` instead of the built in indexing (since there's no default value specified). Also, would it flow better if the two conditional statements were combined?

```
else if (
  event.httpMethod === 'GET' &&
  event.pathParameters &&
  event.pathParameters.executionName
) {
  return get(event, cb);
}
```